<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FanKuiJieGuo_Form</class>
 <widget class="QWidget" name="FanKuiJieGuo_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>同步课堂</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images/images.qrc">
    <normaloff>:/new/prefix1/Images/logo.png</normaloff>:/new/prefix1/Images/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#frame
{
	background-color: rgb(70, 140, 255);
	border-radius:10px;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>400</width>
     <height>450</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>450</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>450</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_stuimange">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_f">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/Images/f.png);
color: rgb(255, 170, 0);	</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_stuname">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>170</y>
      <width>291</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
	color: rgb(255, 255, 255);
	
	font: 500 17pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>401</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
 border-top-left-radius:0px;
 border-bottom-left-radius:10px;
 border-top-right-radius:0px;
 border-bottom-right-radius:10px;
</string>
    </property>
    <widget class="QPushButton" name="pushButton_type">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_f_type">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请求</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Images/images.qrc"/>
 </resources>
 <connections/>
</ui>
